src/config.rs

```diff
@@ -6,7 +6,9 @@ pub use crate::wrap::OutputWrap;
 
 #[derive(Debug, Clone, Copy, PartialEq)]
 pub enum PagingMode {
+    #[cfg(feature = "paging")]
     Always,
+    #[cfg(feature = "paging")]
     QuitIfOneScreen,
     Never,
 }
```

Comment:
Is there a reason for leaving the `PagingMode` enum available (with a single variant) in case `paging` is not enabled? I'm assuming there would be too many changes all over the code base if `PagingMode` would not be available?

Reply:
It's a little more invasive in src/controller.rs but not bad, I've made that change. Good call.